api/v1alpha1/hostedcluster_types.go

```diff
@@ -524,6 +527,12 @@ type PlatformSpec struct {
 
 	// Azure defines azure specific settings
 	Azure *AzurePlatformSpec `json:"azure,omitempty"`
+
+	// IBMCloud defines IBMCloud specific settings for components
+	//
+	// +optional
+	// +immutable
+	PowerVS *PowerVSPlatformSpec `json:"powervs,omitempty"`
```

Comment:
Should PowerVS belong to
```
// IBMCloudPlatformSpec defines IBMCloud specific settings for components
type IBMCloudPlatformSpec struct {
	// ProviderType is a specific supported infrastructure provider within IBM Cloud.
	ProviderType configv1.IBMCloudProviderType `json:"providerType,omitempty"`
         IBMCloudPlatformSpec
}
```

i.e isn't it a ProviderType?

cc @relyt0925

Reply:
We are considering PowerVS as a different platform all together in the OCP project as of now, not sure if we can consider here as a provider type under `IBMCloud`

Reply:
It needs to be separate from IBM Cloud